appeals/api/src/server/infrastructure/topics.js

```diff
@@ -1,6 +1,7 @@
 export const producers = {
 	boCaseData: 'appeal-bo-case',
-	boDocument: 'appeal-bo-document'
+	boDocument: 'appeal-bo-document',
+	boSvcUsr: 'appeal-bo-svcusr'
```

Comment:
As above, I think this abbreviation could potentially be confusing - could it be renamed to `boServiceUser: 'appeal-bo-serviceuser'`?

Reply:
I can rename the key, but the value is the actual service bus topic so need to stick to it.